docs/griptape-framework/data/artifacts.md

```diff
@@ -1,3 +1,8 @@
+---
+search:
+  boost: 2 
+---
```

Comment:
No way to do this per-directory or something?

Reply:
Not as far as I can tell :(

The other solution is to de-rank the content we don't want instead of boosting the content we do. But this did not work well in my experience.